sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -1723,6 +1724,6 @@ public void setHealthConditions(HealthConditionsDto healthConditions) {
 
 	@Override
 	public String toString() {
-		return this.getUuid() + " - " + this.getExternalID();
+		return CaseDataDto.I18N_PREFIX + StringUtils.SPACE + this.getUuid() + " - " + this.getExternalID();
```

Comment:
Just realised that there's a problem with this because cases might not have an external ID; nothing introduced by you, but can you add an `isBlank` check before appending the dash and the external ID?